### ISSUE TITLE
minor - gcs_prefix inconsistent with readme.md

### ISSUE THREAD

**Issue by DevanM** [blockchain-etl/blockchain-etl-streaming#15]
GCS_PREFIX in example_values/ethereum/trace_data/values.yaml inconsistent w/ readme step 4 (ethereum-etl/streaming/last_synced_block.txt)

GCS_PREFIX should read ethereum-etl/streaming

**Comment by voron** [comment]
Yes, readme doesn't cover how to proceed with [separate] traces streaming. We need to run one more etl-streaming instance to parse traces, and it stores it's state into separate `last_synced_block.txt` under `gs://<YOUR_BUCKET_HERE>/ethereum-etl/streaming-trace`. Thus, `trace_data/values.yaml` is correct, as `GCS_PREFIX` has to be different with `block_data/values.yaml`.
